examples/mobile/HomeApp/js/app.js

```diff
@@ -125,6 +125,24 @@ import Storage from "./clipping-storage.js";
 				(app2.isActive) ? 0 : -1 : 1
 		});
 
+		// check webclips order in app
+		appsList = appsList.forEach((app) => {
+			fetch(`${app.webClipsList[0].url}/manifest.json`)
+				.then((data) => data.json())
+				.then((manifest) => {
+					let temp;
+
+					if (manifest.cardType == "service") {
```

Comment:
If you had previously stored the data in an array, now you could use it in `.sort()` method for array